app/containers/message/Components/RightIcons/ReadReceipt.tsx

```diff
@@ -6,15 +6,11 @@ import { useTheme } from '../../../../theme';
 
 const ReadReceipt = React.memo(({ isReadReceiptEnabled, unread }: { isReadReceiptEnabled?: boolean; unread?: boolean }) => {
 	const { colors } = useTheme();
+	const iconName = !unread && unread !== null ? 'check-double' : 'check';
+	const iconColor = !unread && unread !== null ? colors.fontInfo : colors.fontAnnotation;
```

Comment:
```
!unread && unread !== null
```
This is awkward.
Is it this way because unread can be undefined?
If so, you can invert the double negative:

```
const iconName = unread ? 'check' : 'check-double';
```